sci-libs/sundials/sundials-5.1.0.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR="emake"
+FORTRAN_NEEDED=fortran
+FORTRAN_STANDARD="77 90"
+# if FFLAGS and FCFLAGS are set then should be equal
+
+inherit cmake-utils fortran-2 toolchain-funcs
```

Comment:
Also sorry for not noticing it earlier, but you should attempt to use cmake.eclass instead here.

Reply:
I shall replace cmake_utils functions with cmake functions next rebase after decision of klu_lib path variable warning

Reply:
Done